release/src/main/scripts/build_release_candidate.sh

```diff
@@ -205,7 +205,7 @@ if [[ $confirmation = "y" ]]; then
   sha512sum -c apache-beam-${RELEASE}.tar.gz.sha512
 
   echo "-----Signing Source Release apache-beam-${RELEASE}.tar.gz-----"
-  gpg --local-user ${SIGNING_KEY} --armor --detach-sig apache-beam-${RELEASE}.zip
+  gpg --local-user ${SIGNING_KEY} --armor --detach-sig apache-beam-${RELEASE}.tar.gz
```

Comment:
I kept .zip but also added .taz.gz  . Can only one type of source archive exist?
In this particular line I change it because .tar.gz should be here as it is in echo above, but .zip was signed twice.

Reply:
https://pypi.org/project/apache-beam/#files also stores only 'zip' version of sources. I don't see a reason to add a .tar.gz artifact if the content is the same.